### DISCUSSION TITLE
Obsidian Web has dissconect with app

### DISCUSSION THREAD

**Discussion by Velioz** [coddingtonbear/obsidian-web#189] [Bug Report]
### Describe the bug

Hi. decide to install your app in chrome and get some problem. 
![image](https://github.com/coddingtonbear/obsidian-web/assets/167547219/f1e3fce0-a385-4bd6-9787-be5efb9b4516)
Write that connection good but or api server in app or Obsidian Web crush and i take this message.
![image](https://github.com/coddingtonbear/obsidian-web/assets/167547219/a86c4f4e-49ae-49e0-9d28-7c94cb025ae9)
I can connect only if i F5 on page of Obsidian Web, but it happen only for 5 sec then crush again until F5 Obsidian Web again.
![image](https://github.com/coddingtonbear/obsidian-web/assets/167547219/d8ec5e77-946c-4a79-b63a-bf2288a47f0d)
![image](https://github.com/coddingtonbear/obsidian-web/assets/167547219/8e5111a5-4e99-4bdc-a200-76c86dadb59e)
![image](https://github.com/coddingtonbear/obsidian-web/assets/167547219/0a0098aa-24e0-4124-ad91-75367597d4c9)

### To Reproduce

2. Click on '...'
3. Scroll down to '...'

### Operating System

win 10

### Browser

chrome

### Browser Version

124.0.6367.61

**Comment by coddingtonbear** [comment]
I'm afraid I can't say much without you reproducing the "Cannot connect" error and then going to the extension options page and clicking the "Bug Report Export" button in the upper-right corner.

**Reply by Velioz** [reply]
Done
[obsidian-local-rest-api.bug_report (1).json](https://github.com/coddingtonbear/obsidian-web/files/15059717/obsidian-local-rest-api.bug_report.1.json)

**Reply by Velioz** [reply]
Can you understand something ?

**Reply by Velioz** [reply]
Try on another machine with same configuration, problem not dissapear. Win 10 Chrome 124.0.6367.61. First 5 second after refresh site with config it work, then again disconnect.
![1](https://github.com/coddingtonbear/obsidian-web/assets/167547219/5bf17ec3-608b-4f36-8bda-ad381557bd64)
![2](https://github.com/coddingtonbear/obsidian-web/assets/167547219/854c0c3d-de9e-4d86-bc2a-451372a597d0)
![3](https://github.com/coddingtonbear/obsidian-web/assets/167547219/3bbb5b27-ea27-4777-ad15-a788a403215a)

**Reply by Velioz** [reply]
[obsidian-local-rest-api.bug_report.json](https://github.com/coddingtonbear/obsidian-web/files/15075891/obsidian-local-rest-api.bug_report.json)

**Comment by coddingtonbear** [comment]
Sorry -- this is just a side-project for me; so there might be a few days delay between messages because I don't always have free time.

My first guess is that maybe you have a certificate problem -- this sounds a lot bit like the problem that happens if you have an obsolete certificate installed, but it could also be that you have multiple certificates installed (& one of them is obsolete).  What I'd recommend is trying to follow these troubleshooting instructions I wrote a while back: https://github.com/coddingtonbear/obsidian-web/discussions/174#discussioncomment-8740665.

**Comment by Velioz** [comment]
I think deleting Obsidian cert help me. Only after reboot device. Now good. Thx a lot

**Comment by coddingtonbear** [comment]
Awesome; i’m glad you got it sorted out!
